### ISSUE TITLE
Raw download from SSP Database can't be read

### ISSUE THREAD

**Issue by znicholls** [IAMconsortium/pyam#57]
I think this is related to issue #40 

If you download data directly from the [SSP Database](https://tntcat.iiasa.ac.at/SspDb/dsd?Action=htmlpage&page=series) e.g.
[iamc_db_emissions_CO2.xlsx](https://github.com/IAMconsortium/pyam/files/1857947/iamc_db_emissions_CO2.xlsx), then try to read it in you get the following error

```python
xlsx_file_name = os.path.join('..','data','iiasa-ssp-portal','iamc_db_emissions_CO2.xlsx')
df = pyam.IamDataFrame(data=xlsx_file_name)
INFO:root:Reading `../data/iiasa-ssp-portal/iamc_db_emissions_CO2.xlsx`
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-8-af699d821ed6> in <module>()
      1 xlsx_file_name = os.path.join('..','data','iiasa-ssp-portal','iamc_db_emissions_CO2.xlsx')
----> 2 df = pyam.IamDataFrame(data=xlsx_file_name)

/home/zebedee/anaconda2/lib/python2.7/site-packages/pyam/core.pyc in __init__(self, data, **kwargs)
     57             self.data = read_ix(data, **kwargs)
     58         else:
---> 59             self.data = read_files(data, **kwargs)
     60 
     61         # define a dataframe for categorization and other metadata indicators

/home/zebedee/anaconda2/lib/python2.7/site-packages/pyam/utils.pyc in read_files(fnames, *args, **kwargs)
    116         logger().info('Reading `{}`'.format(fname))
    117         df = read_pandas(fname, *args, **kwargs)
--> 118         dfs.append(format_data(df))
    119 
    120     return pd.concat(dfs)

/home/zebedee/anaconda2/lib/python2.7/site-packages/pyam/utils.pyc in format_data(df)
    135         df = pd.melt(df, id_vars=IAMC_IDX, var_name='year',
    136                      value_vars=numcols, value_name='value')
--> 137     df['year'] = pd.to_numeric(df['year'])
    138 
    139     # drop NaN's

/home/zebedee/anaconda2/lib/python2.7/site-packages/pandas/core/tools/numeric.pyc in to_numeric(arg, errors, downcast)
    131             coerce_numeric = False if errors in ('ignore', 'raise') else True
    132             values = lib.maybe_convert_numeric(values, set(),
--> 133                                                coerce_numeric=coerce_numeric)
    134 
    135     except Exception:

pandas/_libs/src/inference.pyx in pandas._libs.lib.maybe_convert_numeric()

ValueError: Unable to parse string "notes" at position 6941
```

`pandas` can read the files fine so I think it's something in the `format_data` method

```python
pd.read_excel(xlsx_file_name).head()
	Model	Scenario	Region	Variable	Unit	2005	2010	2020	2030	2040	2050	2060	2070	2080	2090	2100	Notes
0	AIM/CGE	SSP1-26	R5.2ASIA	Emissions|CO2	Mt CO2/yr	11425.5834	11634.7357	13065.2252	10592.7347	6659.8390	4741.0358	3034.2767	1420.4712	677.9487	129.9080	-108.3545	NaN
1	AIM/CGE	SSP1-34	R5.2ASIA	Emissions|CO2	Mt CO2/yr	11425.5834	11634.7357	13065.2252	12529.0331	11158.2383	9226.0869	6898.9745	5051.7566	3835.2232	2832.4870	2140.2139	NaN
2	AIM/CGE	SSP1-45	R5.2ASIA	Emissions|CO2	Mt CO2/yr	11425.5834	11634.7357	13089.8659	13477.6715	13734.3888	12765.2632	11071.2456	9231.4057	7124.6789	5455.4447	4253.2434	NaN
3	AIM/CGE	SSP1-Baseline	R5.2ASIA	Emissions|CO2	Mt CO2/yr	11425.5834	11659.0560	13213.7447	14956.9321	15380.0704	14824.3021	13783.3527	11478.1689	9623.0991	8231.9889	7424.4252	NaN
4	AIM/CGE	SSP2-26	R5.2ASIA	Emissions|CO2	Mt CO2/yr	11425.5834	12507.0952	15734.6777	10866.7656	6753.7836	3843.2592	2228.3360	524.0113	-266.0716	-877.6898	-1315.4186	NaN
```

```python
pd.read_excel(xlsx_file_name).tail()

Model | Scenario | Region | Variable | Unit | 2005 | 2010 | 2020 | 2030 | 2040 | 2050 | 2060 | 2070 | 2080 | 2090 | 2100 | Notes
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
WITCH-GLOBIOM | SSP5-34 | World | Emissions\|CO2 | Mt CO2/yr | 31922.044354 | 35303.38776 | 37813.933396 | 18924.216520 | 19575.671788 | 18596.184683 | 18180.643978 | 17830.551070 | 17627.297040 | 15580.003928 | 5355.704367 | NaN
WITCH-GLOBIOM | SSP5-45 | World | Emissions\|CO2 | Mt CO2/yr | 31922.044354 | 35303.38776 | 39879.429728 | 32182.096960 | 32652.594997 | 32221.855535 | 31312.519330 | 29779.858417 | 27365.243339 | 24872.151035 | 14496.015076 | NaN
WITCH-GLOBIOM | SSP5-60 | World | Emissions\|CO2 | Mt CO2/yr | 31922.044354 | 35303.38776 | 40954.058910 | 45047.594450 | 51485.070329 | 50229.822011 | 45199.301594 | 45806.835461 | 46193.419795 | 44540.537803 | 38631.339460 | NaN
WITCH-GLOBIOM | SSP5-Baseline | World | Emissions\|CO2 | Mt CO2/yr | 31922.044354 | 35303.38776 | 44321.530964 | 56625.698727 | 72881.159836 | 88326.222670 | 100068.273329 | 109388.272393 | 116226.693340 | 118505.891175 | 114622.309010 | NaN
© SSP Public Database (Version 1.1) https://tn... | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN | NaN
```

**Comment by danielhuppmann** [comment]
thanks @znicholls - indeed, so far we can only process data that is in "clean" IAMC format, so without the Notes column. As a first stab, we could just drop that in the `format_data()` function.

And the last row `@ SSP ...` is added when you download data from the IIASA web databases, so this should also be removed when importing as an `IamDataFrame`.

**Comment by gidden** [comment]
Note, there may also be pandas options for reading in that do this for you,
FYI.

**Comment by danielhuppmann** [comment]
solved by #58
